vowels.c

```diff
@@ -0,0 +1,65 @@
+/*Ashank Kumar, takes the vowels out of a string and prints the new string*/
+#include <stdio.h>
+#include <string.h>
+
+int main()
+{
+	char string[50];
+	char change[50];
+	int counter = 0;
+
+	printf("Enter a string:");
+	fgets(string, 50, stdin);
+
+	string[strlen(string)-1] = '\0';
+
+	/*for(int x = 0; string[x] != '\0'; x <strlen(string);x++)
+	{
+		switch(string[x])
+		{
+			case 'a':
+				break;
+			case 'e':
+				break;
+			case 'i':
+				break;
+			case 'u':
+				break;
+			case 'o':
+				break;
+			case 'A':
+				break;
+			case 'E':
+				break;
+			case 'I':
+				break;
+			case 'U':
+				break;
+			case 'O':
+				break;
+			default:
+				change[counter] = string[x];
+				counter++;
+		}
+	} */
+	
+	for(int x = 0; string[x] != '\0'; x++)
+	{
+		if(string[x] == 'e' || string[x] == 'u' || string[x] == 'o' || string[x] == 'a' || string[x] == 'i' || string[x] == 'E' || string[x] == 'U' || string[x] == 'O' || string[x] == 'A' || string[x] == 'I')
+		{
+			continue;
+		}
+		else
+		{
+			change[counter] = string[x];
+			counter++;
+		}
+			
+	}
+
```

Comment:
So the random symbols printed out because of the following: you're just missing one line that would make your line perfect! Remember that a string has to have a null terminator at the end! because you forgot to make the last element one, it printed out those junk symbols! 

So add `change[counter] = '\0';` and your problem goes away!